### PR TITLE
Add exception if gem is not found

### PR DESCRIPTION
**Pull request by alexeyschepin**
Based on https://github.com/railsbump/app/issues/59

`lockfile`  upload fils if at least one gem is not found. I think such gems must be rather ignored.

The solution is not tested because I wasn't able to run the app locally.

### CODE REVIEW

File: app/services/gemmies/create.rb [hunk 1]
```diff
@@ -12,6 +12,20 @@ def initialize(gemmy)
       end
     end
 
+    class NotFound < Error
+      attr_reader :gemmy_name
```
Comment (manuelmeurer) [inline app/services/gemmies/create.rb:16]: I'd prefer to store `gemmy` here instead of `gemmy_name`, just like `AlreadyExists` does.

**Comment by manuelmeurer** [comment]
Thanks a lot, @alexeyschepin!
Could you address the comment I made?

**Comment by alexeyschepin** [comment]
@manuelmeurer I've made a new commit.

**Comment by manuelmeurer** [comment]
Thanks @alexeyschepin and sorry for the delay!

I'm not sure why I asked you to change it so that `gemmy` is passed to the error instead of `gemmy_name`, it doesn't make sense since, well, there is no gemmy. 🤦‍♂️

I merged anyways and will undo the last commit.

**Pull request merged**